core/src/main/java/io/questdb/cairo/O3Utils.java

```diff
@@ -49,16 +49,20 @@ public static void setupWorkerPool(
         final MessageBus messageBus = cairoEngine.getMessageBus();
         final int workerCount = workerPool.getWorkerCount();
         final O3PartitionPurgeJob purgeDiscoveryJob = new O3PartitionPurgeJob(messageBus, workerPool.getWorkerCount());
-        final ColumnPurgeJob columnPurgeJob = new ColumnPurgeJob(cairoEngine, functionFactoryCache);
 
-        workerPool.assign(purgeDiscoveryJob);
-        workerPool.assign(columnPurgeJob);
+        // ColumnPurgeJob has expensive init (it creates a table), disable it in some tests.
+        if (!cairoEngine.getConfiguration().disableColumnPurgeJob()) {
+            final ColumnPurgeJob columnPurgeJob = new ColumnPurgeJob(cairoEngine, functionFactoryCache);
+            workerPool.assign(purgeDiscoveryJob);
```

Comment:
should not be here

Reply:
fixed in [45c5584](https://github.com/questdb/questdb/pull/2940/commits/45c558456860a7bc84357841539627f4321623e5)